src/lib/Scenes/Home/Components/Sales/Components/ZeroState/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import { WebView } from "react-native"
+
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
+
+export class ZeroState extends React.Component<null> {
+  render() {
+    return (
+      <WebView
+        onShouldStartLoadWithRequest={e => {
+          if (e.navigationType === "click") {
```

Comment:
This captures clicks in the `WebView` to let SwitchBoard handle opening the right view controller